mino/option_test.go

```diff
@@ -33,20 +33,44 @@ func TestFilter_RotateFilter(t *testing.T) {
 
 func TestFilter_IndexFilter(t *testing.T) {
 	filters := &Filter{Indices: []int{}}
-
 	IndexFilter(1)(filters)
-	require.Equal(t, filters.Indices, []int{1})
+	require.Equal(t, []int{1}, filters.Indices)
+
+	IndexFilter(2)(filters)
+	require.Equal(t, []int{1, 2}, filters.Indices)
 
 	IndexFilter(2)(filters)
-	require.Equal(t, filters.Indices, []int{1, 2})
+	require.Equal(t, []int{1, 2}, filters.Indices)
```

Comment:
duplicate w/ lines above?

Reply:
No, this is testing that if you add `2` twice, it only shows up once. But I removed the lines at the end of the test which tried to add `1, 2, 3` to a filter which already has the three indexes.